x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/create_threat_signals.ts

```diff
@@ -113,11 +117,6 @@ export const createThreatSignals = async ({
 
   logger.debug(buildRuleMessage(`Total indicator items: ${threatListCount}`));
 
-  const threatListConfig = {
```

Comment:
If we now have two different values for `threatListConfig`, can we give them distinguishing names? It's not obvious from this level how the results of `getThreatListConfig()` differ from the manually `map`ped version.